src/submission/SubmitBundle.jsx

```
@@ -92,6 +92,17 @@ function SubmitSelection(selection, framesPerTask) {
     var dependencies = findJobAttachments(rqi.comp); // list of filenames
     var compName = dcUtil.removeIllegalCharacters(rqi.comp.name);
 
+    if (app.fonts.missingOrSubstitutedFonts != "") {
+        var submitConfirmation = confirm(
```

Just tried submitting a job that had no missing or substituted fonts, all using just Amazon Ember and the GUI submitter CLI command was not reached and the submitter fails silently.

It's because this variable is never created and set to true if there are no missing fonts. As a result, it defaults to false, returns, and then there's not notification to the customer. This needs to be set to default true initially and then have the if-check with confirmation.

---

Tried to next the second if statement under the first but that didn't work, AE ends up loading and then crashes from not responding.

---

Resolved in ea25a1f